components/widgets/inputs/input/input.tsx

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Moving https://github.com/mattermost/mattermost-webapp/blob/master/components/input.tsx to widgets and applying some enhancements required.
TODO: Migrate to func component.

Reply:
So this new component is based on https://github.com/mattermost/mattermost-webapp/blob/master/components/input.tsx? Once complete, will we remove the original one and update the other components to use this new one?

Reply:
Correct, that's the idea.